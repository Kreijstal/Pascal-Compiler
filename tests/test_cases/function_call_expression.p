program TestFunctionReturnValue;
var
  assignedValue: integer;
function AddTen(val: integer): integer;
begin
  AddTen := val + 10;
end;
begin
  assignedValue := AddTen(5);
  writeln('Assigned value: ', assignedValue);
  writeln('Arithmetic expression: ', 100 + AddTen(20));
  writeln('Direct parameter: ', AddTen(32));
end.
