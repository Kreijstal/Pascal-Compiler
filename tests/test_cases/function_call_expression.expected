Assigned value: 15
Arithmetic expression: 130
Direct parameter: 42
